app/src/main/java/com/duckduckgo/app/browser/tabpreview/WebViewPreviewGenerator.kt

```diff
@@ -33,13 +33,17 @@ interface WebViewPreviewGenerator {
 class FileBasedWebViewPreviewGenerator(private val dispatchers: DispatcherProvider) : WebViewPreviewGenerator {
 
     override suspend fun generatePreview(webView: WebView): Bitmap {
-        disableScrollbars(webView)
-        val fullSizeBitmap = createBitmap(webView)
+        val scaledBitmap: Bitmap
```

Comment:
nit: don't think you need to have `val scaledBitmap: Bitmap` and `return scaledBitmap` separately; could just make it part of the `return` statement at the end of the `try` block. e.g..

```
try {
    disableScrollbars(webView)
    val fullSizeBitmap = createBitmap(webView)

    val scaledHeight = webView.context.resources.getDimension(R.dimen.gridItemPreviewHeight).toPx()
    val scaledWidth = scaledHeight / fullSizeBitmap.height * fullSizeBitmap.width
    return scaleBitmap(fullSizeBitmap, scaledHeight.roundToInt(), scaledWidth.roundToInt())
} finally {
    enableScrollbars(webView)
}
```

Reply:
Good call, thanks!